components/clm/src/biogeophys/SoilHydrologyMod.F90

```diff
@@ -1489,7 +1538,7 @@ subroutine Drainage(bounds, num_hydrologyc, filter_hydrologyc, num_urbanc, filte
                 dzsum  = dzsum + dzmm(c,j)
                 icefracsum = icefracsum + icefrac(c,j) * dzmm(c,j)
              end do
-             ! add ice impedance factor to baseflow
+             ! add ice impedance factor to baseflow REMOVED BY TAO - ICE NOT AN IMPEDENCE IN SALTWATER SYSTEMS
```

Comment:
We got around this by changing the freezing pt to reduce the amount of ice in the system that could impede flow

Reply:
```suggestion
             ! add ice impedance factor to baseflow
```